iwallet/balance.go

```diff
@@ -16,20 +16,23 @@ package iwallet
 
 import (
 	"fmt"
+
 	"github.com/spf13/cobra"
 )
 
-// accountInfoCmd represents the balance command
+// accountInfoCmd represents the balance command.
 var accountInfoCmd = &cobra.Command{
 	Use:   "balance",
-	Short: "check information of specified account",
-	Long:  `check information of specified account`,
-	RunE: func(cmd *cobra.Command, args []string) (err error) {
```

Comment:
The `balance` command is show account's all information, not just the balance.
@lispc Do you have any suggestions?

Reply:
I c